### ISSUE TITLE
[Not fixed in #11637] Missing focus management and narration on PDP when 'Show Reviews' is clicked

### ISSUE THREAD

**Issue by BenjaminEmiliani** [SAP/spartacus#14030]
Address after #11637 

**Describe the bug**
For a11y users, there is not focus shift or vocalized feedback when users select 'Show reviews' link on a product display page.

**Tell us the version of Spartacus**
- 4.1

**To Reproduce**
1. Click on any product with reviews
2. Click on 'Show reviews'
4. Observe current focus and lack of context feedback

**Expected behavior**
Focus shifts to 'Reviews' tab. 
vocalized feedback that user is in the Reviews section

**Comment by saad-mo** [comment]
@developpeurweb any suggestions on how to 'vocalized feedback that user is in the Reviews section'

**Comment by developpeurweb** [comment]
@saad-mo once ticket GH-11637 is finished we're expecting the new component to provide context, such as _"tab group current tab, reviews"_.

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using yarn
More extensive testing (on other screenreaders and devices) will be done with RC.
<img width="1189" alt="Screen Shot 2022-04-13 at 1 41 02 PM" src="https://user-images.githubusercontent.com/45409515/163239315-bc66edbb-d251-47be-97ef-967d00826c2c.png">
![reviewsTab](https://user-images.githubusercontent.com/45409515/163239349-cfce38ba-b180-4b6f-8621-e25b37fc3875.png)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using yarn
More extensive testing (on other screenreaders and devices) will be done with RC.

<img width="1189" alt="Screen Shot 2022-04-13 at 1 41 02 PM" src="https://user-images.githubusercontent.com/45409515/163239315-bc66edbb-d251-47be-97ef-967d00826c2c.png">


![reviewsTab](https://user-images.githubusercontent.com/45409515/163239471-f1ad4600-b2be-46a5-be30-ee980c7888ca.png)
